### PR TITLE
Update UI for campaign short-link page

### PR DESCRIPTION
**Pull request by Jeremy-D**

### CODE REVIEW

**Comment by Jeremy-D** [comment]
Updated UI pictured below

Screen Shot 2019-05-13 at 8 19 41 PM

Screen Shot 2019-05-13 at 8 19 55 PM

File: app/views/campaign.hbs [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<style>
+  body {
+  background-color: white;
+}
+</style>
```
Comment (seport) [inline app/views/campaign.hbs]: can we move this into a stylesheet plz 🙏

File: public/scss/campaign.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@
     border: 1px solid #FFF;
     margin: 25px 0;
     border-radius: 2.5px;
-    padding: 20px 10px;
+    /*padding: 20px 10px;*/
```
Comment (seport) [inline public/scss/campaign.scss]: unnecessary comment, plz delete

**Comment by seport** [comment]
F

**Pull request closed without merging**